### ISSUE TITLE
How do we establish an intital price for our coin?

### ISSUE THREAD

**Issue by bennytehcat** [The-GNTL-Project/Exchange#53]
We need to find the ideal price range for an initial price. Open to suggestions here, but we can't just open with a blank slate.  The current proposed price is 1p. 

Total circulation will be 77.7M GNTL.  I estimate that if we were to go live in 6 months, there would be ~7M GNTL in circulation, or 10% of the total supply.  A valuation of 1p puts a total valuation of $70k in active coin, $777k if it remains flat.  

1p = 19 satoshi (April 29, 2021)

19 sat opening price leaves little room for a floor dropping out and flat-lining to zero.  However, I think to even use 1p (or to suggest a higher value), we need to justify a $70k (or higher) value.  This is going to be a challenge we all need to consider -- placing an intrinsic value on our coin.

**Comment by tylerjohneddy** [comment]
personally i think that the inital price should be set as low as possible to start with to stop people trying to dump all there coins as soon as its listed

not sure what @BKdilse has to add to this

**Comment by BKdilse** [comment]
With coins getting listed, it's likely to be a dump, but of course that requires buys.  Once we have more value, price should increase.

**Comment by BKdilse** [comment]
Linked to [#51 ](url)

**Comment by Eadrom** [comment]
Preseeding some orders like on #51 at price points set from the current trading going on the tipbot exchange should provide a decent starting point and helps solve the problem of picking a starting valuation for GNTL. Those orders are basically what the community values GNTL at and were created fairly organically.

**Comment by baldtrooper** [comment]
If we go onto an already established exchange do we not just start at 1 sat and then the market decides from there? As you say though Eadrom the tipbot trading bot has already established a rough price, my 3 sat orders (approx) are selling and I've now put a couple on for 4 sats (ish). I do think we would quickly move over 10 sats on a larger exchange, you only need a handful of buy orders to get this level.

**Comment by BKdilse** [comment]
True, the price will be set by the traders at the end of the day, what they are willing to sell at.

**Comment by tylerjohneddy** [comment]
on another exchange we have a start price
